Analysis/include/EventInfo.h

```diff
@@ -210,6 +237,42 @@ class EventInfoBase {
         return selected_jets;
     }
 
+    static JetPair SelectVBFJetPair(const JetCollection& jets_vbf)
+    {
+        double max_mjj = -std::numeric_limits<double>::infinity();
+        JetPair selected_pair = ntuple::UndefinedJetPair();
+        for(size_t n = 0; n < jets_vbf.size(); ++n) {
+            for(size_t h = n+1; h < jets_vbf.size(); ++h) {
+                const JetCandidate& jet_1 = jets_vbf.at(n);
+                const JetCandidate& jet_2 = jets_vbf.at(h);
+                const LorentzVector jet_12 = jet_1.GetMomentum() + jet_2.GetMomentum();
+                if(jet_12.M() > max_mjj){
+                    max_mjj = jet_12.M();
+                    if(jet_1.GetMomentum().Pt() > jet_2.GetMomentum().Pt())
+                        selected_pair = std::make_pair(n,h);
+                    else
+                        selected_pair = std::make_pair(h,n);
+                }
+            }
+        }
+
+        return selected_pair;
+    }
+
+    double GetHTotherJets()
```

Comment:
Rename function to GetHT and add bool includeHbbJets. Modify the code accordingly.